src/hooks/use-extended-theme/types.ts

```diff
@@ -0,0 +1,13 @@
+import type { stardust } from '@nebula.js/stardust';
+
+export interface BackgroundColors {
+  tableColorFromTheme: string;
+  color?: string;
+  isDark: boolean;
+  isTransparent: boolean;
+}
+
+export interface ExtendedTheme extends stardust.Theme {
```

Comment:
The API for now remains the same as in the `sn-table`. Note that some of the background props are not all that relevant for the `sn-pivot-table`.